CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Unreleased
 ----------
+
+22.00.1 (March 12, 2024)
```

Comment:
Just wondering - is the double `00` expected? I don't think it matters but just curious why it would let that happen :)

Reply:
No 😞

Reply:
@bjk